### PR TITLE
Accessibility Improvements for Application Passwords #51580

### PR DESCRIPTION
**Pull request by georgestephanis**
WIP improvements mitigating feedback for Application Passwords.

Trac ticket: https://core.trac.wordpress.org/ticket/51580#ticket

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-admin/css/forms.css [hunk 1]
```diff
@@ -854,7 +854,7 @@ table.form-table td .updated p {
 	margin-bottom: 0;
 }
 
-.new-application-password-notice.notice input.code {
+.password-display input.code {
```
Comment (TimothyBJacobs) [inline src/wp-admin/css/forms.css:857]: Thinking this selector might be a bit generic to have in a global css file.

Comment (georgestephanis) [reply src/wp-admin/css/forms.css]: Further namespaced it via https://github.com/WordPress/wordpress-develop/pull/638/commits/87550e27fa227c8b248489bea6f14a31700cb12a

Comment (georgestephanis) [reply src/wp-admin/css/forms.css]: Or would you rather have an extra class on the input?

Comment (TimothyBJacobs) [reply src/wp-admin/css/forms.css]: That fix looks good to me.

File: src/wp-admin/includes/class-wp-application-passwords-list-table.php [hunk 2]
```diff
@@ -105,8 +105,17 @@ public function column_last_ip( $item ) {
 	 *
 	 * @since 5.6.0
 	 */
-	public function column_revoke() {
-		submit_button( __( 'Revoke' ), 'delete', 'revoke-application-password', false );
+	public function column_revoke( $item ) {
```
Comment (TimothyBJacobs) [inline src/wp-admin/includes/class-wp-application-passwords-list-table.php:108]: We need to add an `@param` document.

**Comment by TimothyBJacobs** [comment]
It also looks like each `id` for the revoke button is the same. I think we need to update them to each have a unique name/id.

We also need to update `WP_Application_Passwords_List_Table::print_js_template_row()` to not call `column_revoke()` anymore since it requires an item now. Passing the underscore template to `submit_button()` might work? Otherwise we'll have to build the HTML manually.

File: src/wp-admin/includes/class-wp-application-passwords-list-table.php (hunk 2 above)
Comment (georgestephanis) [reply src/wp-admin/includes/class-wp-application-passwords-list-table.php:110]: Fixed

**Comment by TimothyBJacobs** [comment]
Fixed in 5e31ccfee21547844d889e3475ecf34177874983.

**Pull request closed without merging**